src/status_im/contexts/shell/share/wallet/view.cljs

```diff
@@ -0,0 +1,94 @@
+  (rf/dispatch [:show-bottom-sheet
+                {:theme :dark
+                 :shell? true
+                 :content
+                 (fn []
+                   [network-preferences/view
+                    {:blur?             true
+                     :selected-networks (set @selected-networks)
+                     :on-save           (fn [chain-ids]
+                                          (rf/dispatch [:hide-bottom-sheet])
+                                          (reset! selected-networks (map #(get utils/id->network %)
+                                                                         chain-ids)))}])}]))
+(defn wallet-qr-code-item
+  [account width index]
+  (let [selected-networks (reagent/atom [:ethereum :optimism :arbitrum])
+        wallet-type       (reagent/atom :wallet-legacy)]
+    (fn []
+      (let [share-title         (str (:name account) " " (i18n/label :t/address))
+            qr-url              (utils/get-wallet-qr {:wallet-type       @wallet-type
+                                                      :selected-networks @selected-networks
+                                                      :address           (:address account)})
+            qr-media-server-uri (image-server/get-qr-image-uri-for-any-url
+                                 {:url         qr-url
+                                  :port        (rf/sub [:mediaserver/port])
+                                  :qr-size     qr-size
+                                  :error-level :highest})]
+        [rn/view {:style {:width width :margin-left (if (zero? index) 0 -30)}}
+         [rn/view {:style style/qr-code-container} 
+          [quo/share-qr-code
+           {:type                @wallet-type
+            :qr-image-uri        qr-media-server-uri
+            :qr-data             qr-url
+            :networks            @selected-networks
+            :on-share-press      #(share-action qr-url share-title)
+            :profile-picture     nil
+            :unblur-on-android?  true
+            :full-name           (:name account)
+            :customization-color (:color account)
+            :emoji               (:emoji account)
+            :on-multichain-press #(reset! wallet-type :wallet-multichain)
+            :on-legacy-press     #(reset! wallet-type :wallet-legacy)
+            :on-settings-press   #(open-preferences selected-networks)}]]]
+      ))))
+
+(defn wallet-tab
+  []
+  (let [accounts (rf/sub [:wallet/accounts])
+        width    (rf/sub [:dimensions/window-width])]
+    [rn/view
+     [rn/scroll-view
+      {:horizontal                true
+       :deceleration-rate         0.9
+       :snap-to-alignment         "start"
+       :snap-to-interval          (- width 30)
+       :disable-interval-momentum true
+       :scroll-event-throttle     64
+       }
+      (map-indexed
+       (fn [index account]
+         ^{:key (:address account)}
+         [wallet-qr-code-item account width index])
+       accounts)]]))
```

Comment:
Better use a FlatList here.

Reply:
Hey @OmarBasem : I think its okay to use a scroll view for a list that is not too long.
What do you think will be the maximum number of wallet accounts a user would have?

Reply:
> 

@siddarthkay there isn't currently an upper limit, but realistically around 20 would be the max anybody would have in the worst case.

Reply:
I would have thought given the estimated size of the data `ScrollView` should be suffice. In general I have always used `FlatList` for larger datasets. 

Another practical point is that it is a little bit more difficult to manage state with a `FlatList` as component state is not maintained due to the components unmounting once they are off the screen and being recrated from scratch once displayed again. In this case state is required to display the `Legacy` or `Multichain` address.

If people feel strongly against this I am more than happy to implement using a `FlatList` component.

Reply:
When testing 20 items with `ScrollView` I noticed some performance issues. Will update to use `FlatList`

Reply:
I will do some performance testing with 20 items and compare both approaches side by side.

Reply:
20 items

`FlatList`

https://github.com/status-im/status-mobile/assets/6873736/bbb2ecf7-d5ac-488c-abef-dd861e9c380b


`ScrollView`

https://github.com/status-im/status-mobile/assets/6873736/b27eed41-55de-4c54-9c97-35a04fc54ba1

@OmarBasem @siddarthkay  Looks like there is a noticeable delay using  `ScrollView`, so I will update to use `FlatList`.